implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -124,7 +125,14 @@ async fn retrieve_project(app_state: &AppState, space: &Space) -> Result<Project
             .await
             .map_err(|e| miette!(e))?
     };
-    let project = match projects.iter().find(|p| p.name == *PROJECT_NAME) {
+
+    let email = app_state.user_email().await.unwrap_or_default();
+
+    let project = match projects
```

Comment:
Can this be refactored with [this part](https://github.com/build-trust/ockam/pull/5718/files#diff-fc44c8a5e4b38ca36b52578860084fa796e4931a7b0efdbeb3396c6d36e5ada5R65) as one `get_user_project` command?

Reply:
They're not trivially identical, since one of them wants to find by ID and one wants to find by name. I took a shot at consolidating the shared concerns with 08c4d617acebdf096b0f57209c53194f20516deb...17177e61eb8680bf5a07c7b6fbfebdcc211b7efc.

Reply:
Thanks